usaspending_api/etl/award_helpers.py

```diff
@@ -97,21 +97,33 @@ def update_contract_awards(award_tuple=None):
         sql_txn_contract_latest += 'WHERE award_id IN %s '
```

Comment:
Since potential_total_value_of_award was the only field we were getting from the latest contract txn record, I think we can flat out replace lines 90-98 with the new `sql_txn_totals` common table expression you've added.